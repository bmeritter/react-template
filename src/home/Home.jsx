import React, { Component } from 'react';
import { Calendar } from '../components/calendar/Calendar.jsx';


export default class Home extends Component {
	render() {
		return (
			<div>
				welcome home
				<Calendar />
			</div>
		);
	}
}
