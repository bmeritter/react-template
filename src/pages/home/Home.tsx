import React from 'react';
import styled from 'styled-components';

const StyledDiv = styled.div`
  background-color: #7f7f7f;
  text-align: center;
`;

const StyledText = styled.p`
  color: black; 
`;
const Home = () => {
  return (
    <StyledDiv>
      <StyledText>welcome home</StyledText>
    </StyledDiv>
  );
};

export default Home;
