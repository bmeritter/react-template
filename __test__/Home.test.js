import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../src/pages/home/Home';

describe('About component', () => {
	it('snapshot', () => {
		const comp = renderer.create( <Home />);
		expect(comp).toMatchSnapshot();
	});
});
