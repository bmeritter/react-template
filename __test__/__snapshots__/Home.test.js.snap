// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component snapshot 1`] = `
<div>
  welcome home
</div>
`;
